torch/testing/_internal/common_methods_invocations.py

```diff
@@ -704,6 +704,26 @@ def sample_inputs_trace(self, device, dtype, requires_grad, **kwargs):
                                      low=None, high=None,
                                      requires_grad=requires_grad))),)
 
+
+def sample_inputs_transpose_swapdims(self, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+
+    cases = [((1, 2, 3), (-1, -2)),
+             ((1, 2, 3), (-1, 2)),
+             ((1, 2, 3), (1, -2)),
+             ((1, 2, 3), (1, 2)),
+             ((), (0, 0)),
+             ((1, ), (0, 0)),
+             ((L, L), (0, 1)),
```

Comment:
L, L -> M, M?

Reply:
Updating to (M, M)
**Note**: Single run timings (very high variability)

Timings with (S, S)
```
=================================================== 62 passed, 35 skipped, 14505 deselected, 5 warnings in 23.48s ====================================================
```

Timings with (M, M)
```
==================================================== 62 passed, 35 skipped, 14505 deselected, 5 warnings in 23.21s =====================================================
```

Timings with (L, L)
```
==================================================== 62 passed, 35 skipped, 14505 deselected, 5 warnings in 29.23s =====================================================
```